lib/handlers/router-entities/route-caching/dynamo-route-caching-provider.ts

```diff
@@ -300,8 +303,13 @@ export class DynamoRouteCachingProvider extends IRouteCachingProvider {
       const shouldSendCachingRequest =
         result.Items &&
         (result.Items.length == 0 || // no caching request has been sent recently
-          // or there is not a single sample for the current block
-          result.Items.every((record) => (record.blockNumber ?? 0) < currentBlockNumber))
+          // or every sampled record is older than maximum blocks diff allowed for the chain
+          result.Items.every((record) => {
+            const blocksDiff = currentBlockNumber - (record.blockNumber ?? 0)
+            const maximumBlocksDiff =
+              this.BLOCKS_DIFF_BETWEEN_CACHING_QUOTES.get(partitionKey.chainId) || this.DEFAULT_BLOCKS_DIFF_CACHING
```

Comment:
for now `BLOCKS_DIFF_BETWEEN_CACHING_QUOTES` is always  null so we defualt to `DEFAULT_BLOCKS_DIFF_CACHING` right?

Reply:
that one actually contains Mainnet: 3

Reply:
https://github.com/Uniswap/routing-api/pull/416/files#diff-ba752e09db66ef293edd546c80fadaf47bb160d3542dca3c31cbbb3628bd0cdcR85

Reply:
i cant read lol